File: be_laravel/app/Models/Question.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Question extends Model
{
    // use HasFactory;

    // public $timestamps = true;
    // protected $table = 'questions';
    // protected $primaryKey = 'id_question';

    // protected $fillable = [
    //     'id_question',
    //     'the_question',
    //     'photo_question',
    //     'answer_1',
    //     'answer_2',
    //     'answer_3',
    //     'answer_4',
    // ];

    // public static function getAllData()
    // {
    //     return self::all();
    // }

    // public static function getQuestionById($id)
    // {
    //     return self::find($id);
    // }

    use HasFactory;

    public $timestamps = true;
    protected $table = 'questions';
    protected $primaryKey = 'id_question';

    protected $fillable = [
        'the_question',
        'profile',
        'answers',
    ];

    protected $casts = [
        'answers' => 'json', // Cast the 'answers' attribute to JSON
    ];

    public static function getAllData()
    {
        return self::all();
    }

    public static function getQuestionById($id)
    {
        return self::find($id);
    }
}

File: be_golang/models/question.go
package models

import (
	"encoding/json"
	"time"
)

type Question struct {
	IdQuestion  int             `json:"id_question" gorm:"primaryKey:autoIncrement"`
	TheQuestion string          `json:"the_question" gorm:"type:varchar(255)"`
	Profile     string          `json:"profile" gorm:"type:varchar(255)"`
	Answers     json.RawMessage `json:"answers"`
	CreatedAt   time.Time       `json:"created_at"`
	UpdatedAt   time.Time       `json:"updated_at"`
}
